### PR TITLE
Fix a few small things

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: packages/gatsby/src/utils/webpack.config.js [hunk 1]
```diff
@@ -326,7 +326,7 @@ module.exports = async (
       modulesDirectories: [
         directoryPath(`node_modules`),
         `node_modules`,
-        directoryPath(`node_modules`, `gatsby`, `node_modules`),
+        path.join(__dirname, `..`, `..`, `node_modules`),
```
Comment (jquense) [inline packages/gatsby/src/utils/webpack.config.js]: this also shouldn't look in the gatsby sites `node_modules`, gatsby may not be there (could be hoisted).

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit b40c9bba880187441a76bd836941f9ebd88558fd

https://app.netlify.com/sites/gatsbyjs/deploys/59779ddfcf321c1c38bbdcc3

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit b40c9bba880187441a76bd836941f9ebd88558fd

https://app.netlify.com/sites/image-processing/deploys/59779de0cf321c1c38bbdcd3

**Comment by KyleAMathews** [comment]
This can be closed?

**Pull request closed without merging**